core/src/main/java/org/jahia/modules/modulemanager/rest/filters/ModuleManagerAuthenticationRequestFilter.java

```diff
@@ -55,6 +55,7 @@ public class ModuleManagerAuthenticationRequestFilter implements ContainerReques
     public void filter(ContainerRequestContext requestContext) throws IOException {
         JahiaUser user = JCRSessionFactory.getInstance().getCurrentUser();
         String username = JahiaUserManagerService.GUEST_USERNAME;
+        // toolManager seems to be an old role which is not used anymore. It should be removed once validated.
```

Comment:
`toolManager` was related to the application server roles that are not used anymore so can be removed. 
See last entry in `what's new` section in https://academy.jahia.com/home/downloads/jahia8-release-notes/jahia-801.html#whats-new

Reply:
done